lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -1432,6 +1424,221 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
+                \a18200581c8adbd72dd6b5b46b27df79b1f8bcbcf0ac780d515f5b57cb08a2\
+                \c9a71b000000e8d4a5100082008200581c9c19c3caa333ee30c6d5d9f0ddd0\
+                \1ad09258a47dec0380519bcae7ac82068200a18200581c9c19c3caa333ee30\
+                \c6d5d9f0ddd01ad09258a47dec0380519bcae7ac1b000000e8d4a510008200\
+                \8200581c6433cd346858f15142171023c633ae0646bdc0470de5ae6f110bdf\
+                \0682068200a18200581c6433cd346858f15142171023c633ae0646bdc0470d\
+                \e5ae6f110bdf061b000000e8d4a5100082008200581c63ec6f04e6fa18e830\
+                \05a02bc57d72f7afa3a04523d016010076b40a82068200a18200581c63ec6f\
+                \04e6fa18e83005a02bc57d72f7afa3a04523d016010076b40a1b000000e8d4\
+                \a5100082008200581c990d9d698730cbc4c09f95be75f54e47c524c3e7ad48\
+                \4de626ef321482068200a18200581c990d9d698730cbc4c09f95be75f54e47\
+                \c524c3e7ad484de626ef32141b000000e8d4a5100082008200581ccc8116d5\
+                \0326ea87caa3e46597b54e56725ff1fe39d1bc08361bc20682068200a18200\
+                \581ccc8116d50326ea87caa3e46597b54e56725ff1fe39d1bc08361bc2061b\
+                \000000e8d4a5100082008200581c246a121534ab486f4e47618cb192568b77\
+                \a491cf4db613a80ced4d7682068200a18200581c246a121534ab486f4e4761\
+                \8cb192568b77a491cf4db613a80ced4d761b000000e8d4a510008200820058\
+                \1c36cf17310d216fc7a2ac6122088766bbc5761129b1155d495bf8112b8206\
+                \8200a18200581c36cf17310d216fc7a2ac6122088766bbc5761129b1155d49\
+                \5bf8112b1b000000e8d4a5100082008200581c1bb104a403de68b0c03438a1\
+                \0d9142a595f4a9ff8162b195493bf55082068200a18200581c1bb104a403de\
+                \68b0c03438a10d9142a595f4a9ff8162b195493bf5501b000000e8d4a51000\
+                \82008200581cc2a47d500058e60176c437fc61be7d0cd07f0d8c871abe6d00\
+                \2636a182068200a18200581cc2a47d500058e60176c437fc61be7d0cd07f0d\
+                \8c871abe6d002636a11b000000e8d4a5100082008200581ce8b783e08083c2\
+                \3c2682afcd4b7a5cb0851239e5f9c04beb2455979582068200a18200581ce8\
+                \b783e08083c23c2682afcd4b7a5cb0851239e5f9c04beb245597951b000000\
+                \e8d4a5100082008200581cedf156a660897651bd753aefa7af7f81d6d83dfc\
+                \9bcdc4afbd36fad382068200a18200581cedf156a660897651bd753aefa7af\
+                \7f81d6d83dfc9bcdc4afbd36fad31b000000e8d4a5100082008200581c30c6\
+                \00d4fcf006fc2067721c55fc7d3a696b93a4f382aaad75e3516682068200a1\
+                \8200581c30c600d4fcf006fc2067721c55fc7d3a696b93a4f382aaad75e351\
+                \661b000000e8d4a5100082008200581cc7d5e024d22767a891e02834b27588\
+                \5e06ed04869747cff43cec91e082068200a18200581cc7d5e024d22767a891\
+                \e02834b275885e06ed04869747cff43cec91e01b000000e8d4a51000ff9fff\
+                \f6" :: Text
+
+        let cborHexMIR = fromTextEnvelope cborHex
```

Comment:
I think that instead of showing just `"mir"`, `{"certificate_type": "mir"}` would be more consistent with the others, and more clear.

Also there seems to be a bit more information in the CBOR for mir, perhaps we want to show it too?

```
[6, [0, {[0, h'FB3C13A29D3798F1B77B47F2DDB31C19326B87ED6F71FB9A27133AD5']: 1000000000000}]]
``` 
https://github.com/input-output-hk/cardano-ledger/blob/master/eras/alonzo/test-suite/cddl-files/alonzo.cddl#L170-L176

Reply:
this is tx we send during faucet startup;-) very good thing to decode IMHO. I changed to `{"certificate_type": "mir"}` Feel free to cut cbor if you feel so. We may also stick to it and add additional tests checking order!